lib/services/gitlab_issues.rb

```diff
@@ -321,7 +313,7 @@ def server_display_url
 
   def integrate_release_with_gitlab_milestone(release, milestone)
     api.create_integration_fields("releases", release.reference_num, data.integration_id,
-      {id: milestone['id'], number: milestone['iid'], url: "#{get_project_url}/milestones/#{milestone['iid']}"})
+      {milestone_id: milestone['id'], number: milestone['iid'], url: "#{get_project_url}/milestones/#{milestone['iid']}"})
```

Comment:
Yes and no. There are only 14 existing integrations and about 5 of them are my test integrations, so I'm just going to manually fix affected data after deploying. (I don't see a way around breaking existing ones -- at least not one that's less time-consuming than just fixing the field keys.)